aws/cluster-autoscaler-values.yml

```diff
@@ -4,4 +4,11 @@ rbac:
 cloudProvider: aws
 
 autoDiscovery:
-  enabled: true
\ No newline at end of file
+  enabled: true
+  clusterName: jwebbinar
+  tags:
+  - k8s.io/cluster-autoscaler/enabled
+  - k8s.io/cluster-autoscaler/{{ .Values.autoDiscovery.clusterName }}
+  #roles:
```

Comment:
what's the deal with these comments? should they be removed

Reply:
Those changes were inherited from the as-is state of the jwebbinar debug effort.  @cslocum should probably call whether or not we keep them,  etc.

Reply:
I say they are not needed.